common/src/main/java/com/wynntils/services/mapdata/providers/builtin/CharacterProvider.java

```diff
@@ -71,10 +118,168 @@ public Optional<MapVisibility> getIconVisibility() {
         }
     }
 
+    public static final class HadesPlayerLocation implements MapLocation {
+        private final HadesUser hadesUser;
+
+        public HadesPlayerLocation(HadesUser hadesUser) {
+            this.hadesUser = hadesUser;
+        }
+
+        @Override
+        public Location getLocation() {
+            return new Location((int) hadesUser.getX(), (int) hadesUser.getY(), (int) hadesUser.getZ());
+        }
+
+        @Override
+        public String getFeatureId() {
+            return "player-" + hadesUser.getName().toLowerCase(Locale.ROOT);
+        }
+
+        @Override
+        public String getCategoryId() {
+            return "wynntils:players:hades" + (hadesUser.getRelation().isEmpty() ? "" : ":" + hadesUser.getRelation());
```

Comment:
I don't think we should expose "hades" here. I know we have done so in some other places, and I don't like it. It's an internal-only name for us developers. I think just `wynntils:players` is good enough. Perhaps we should also change so we *always* have a third part, so if we have no known hades relation we should use `wynntils:players:other`. I'm ot sure how it could show up then (perhaps by implementing a "entity radar" function?) but I think it is better to be prepared.

Reply:
Also, I think I tried to always use singular in category ids. If I didn't, we should fix it. It might not feel like the best fit in all situations, but that is basically the only way to be consistent which I think is more important.

Reply:
Okay. I can fix that up.

Reply:
Fixed.